scripts/qap_functional_temporal.py

```diff
@@ -378,6 +378,7 @@ def run(subject_list, config, cloudify=False):
     if config['write_report']:
         import qap.viz.reports as qvr
         from nipype import logging
+	import os.path as op
```

Comment:
Hi @craigmoodie, yes if you look at the code, it seems like you are using a regular tab instead of 4 spaces. This can be configured in python, but it is not accepted by default. It'd be worth looking at your editor settings so that it translates all tabulations to the corresponding spaces. In this piece of code it should have 8 spaces.